x-pack/test/security_solution_cypress/cypress/e2e/investigations/timelines/search_or_filter.cy.ts

```diff
@@ -53,18 +55,17 @@ describe('Timeline search and filters', { tags: ['@ess', '@serverless'] }, () =>
     });
   });
 
-  // FLAKY: https://github.com/elastic/kibana/issues/169882
-  describe.skip('Update kqlMode for timeline', () => {
+  describe('Update kqlMode for timeline', () => {
     beforeEach(() => {
       login();
+      deleteTimelines();
       visit(TIMELINES_URL);
       waitForTimelinesPanelToBeLoaded();
       openTimelineUsingToggle();
       cy.intercept('PATCH', '/api/timeline').as('update');
       cy.get(LOADING_INDICATOR).should('not.exist');
       showDataProviderQueryBuilder();
       cy.get(TIMELINE_SEARCH_OR_FILTER).click();
```

Comment:
what is the point in clicking on a element at the end of a test? I thing we should remove this line. Then I'm not sure of the value of the next line up either...

Reply:
Changed that as part of https://github.com/elastic/kibana/commit/76a9814698b1d07820f94193cb3250d818ef2360